Docs/publish.sh

```diff
@@ -1,8 +1,12 @@
-#!/bin/bash
+#!/bin/ash
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
It's not typo, but means [Almquist Shell][1] (hence `ash`), which an extended variant of POSIX Bourne Shell (`sh`).  I switched from Bourne-Again Shell (`bash`) to `ash` here, for better portability. Some Linux distros like Alpine Linux, which is a de facto standard for Docker images, lacks `bash` by default.

[1]: https://en.wikipedia.org/wiki/Almquist_shell

Reply:
That's good. I didn't know about `ash`.